src/workloads/c2c/ChangeEventApplication.yml

```diff
@@ -13,6 +13,9 @@ Keywords:
 - change event application
 - CEA
 
+EnvironmentVariables:
+  MongosyncConnectionUrls: http://localhost:27182
```

Comment:
Actually, shouldn't this be set inside `mongodb_setup.xxx`file? The same workload should be able to run inside both single or multiple Mongosync instances setup.

Reply:
ah, great. Thanks for the explanation!

Reply:
The connection strings are configured in mongodb_setup here:
https://github.com/10gen/dsi/pull/1118/files#diff-870e9d10c6e89edfd67250b4a3dca1e54fcfb5e2853b2ecaf541c0373499b8a9R120

And then passed to genny in the test_control here:
https://github.com/10gen/dsi/pull/1118/files#diff-371f59723914a0fe0f3fcfafade32f3d0255d7fcce70b8f70648133163cbda28R11

Reply:
Thanks. I see it now. If @huan-Mongo is good with this one we can close this thread.